a1.py

```diff
@@ -293,10 +293,10 @@ def main():
     pv1.addPrescribedItem(p2)
     
     print(cv1)
-    print(cv1.prescribedItemListStr())
+    cv1.prescribedItemListStr()
```

Comment:
The reason this change works is `prescribedItemListStr` itself prints the output, and does not return a value. so when you call print on the method it prints None because there is no return value for `prescribedItemListStr`.

that said you might want to change the method of `prescribedItemListStr` will show this in different change

Reply:
Actually, this should be fine as is, but we might want to consider renaming `prescribedItemListStr` to `printPrescribedItemList`